colossalai_platform/cli/commands/projects/train.py

```diff
@@ -1,38 +1,74 @@
 import os
-from patch import patch_tensorboard
+import argparse
+import types
+from patch import patch_platform_specific_dependencies
 
-# ==================================
-# Code generated by ColossalAI-Platform-CLI. DO NOT EDIT.
+# Please do not remove this call,
+# the platform's runtime environment needs it.
+patch_platform_specific_dependencies()
+
+# ===================================================================
+# Example code generated by ColossalAI-Platform-CLI.
 #
-# ColossalAI Platform Specification
-# Start
-# ==================================
+# Modify it to fit your needs.
+# ===================================================================
 
-# patch dependencies
-patch_tensorboard()
 
-# Environment variables for your data access
-PROJECT_DIR = os.environ['PROJECT_DIR']
-DATASET_DIR = os.environ['DATASET_DIR']
-OUTPUT_DIR = os.environ['OUTPUT_DIR']
-INPUT_MODEL_DIR = os.environ['MODEL_DIR']
-CHECKPOINT_DIR = os.path.join(OUTPUT_DIR, 'checkpoint')
-TENSORBOARD_DIR = os.path.join(OUTPUT_DIR, 'tensorboard')
+def parse_args(parser: argparse.ArgumentParser):
+    # required arguments
+    parser.add_argument(
+        "--epoch",
+        type=int,
+        required=True,
+        help="The training epoch.",
+    )
```

Comment:
`epoch` is not a default arg, we should only handle args which are passed as environment variables by the platform. Some programs do not even have epochs, they just set the number of iterations directly.

Reply:
But the default `Hyperparameter.json` has a `epoch` field.

This argument is not a default one, but to act as an example:

```json
{
  "HyperParameters": [
    {
      "name": "epoch",
      "types": "string",
      "defaultValue": "10"
    }
  ]
}
```